k-distribution/tests/regression-new/map-symbolic-tests-haskell/assignment-10-spec.k.out

```diff
@@ -0,0 +1,28 @@
+  #Not ( {
+    X
+  #Equals
+    Y
+  } )
+#And
+  #Not ( {
+    Y
+  #Equals
+    Z
+  } )
+#And
+  <k>
+    assignmentResult ( ( MAP [ Y:MyId <- 2 ]
+    X:MyId |-> 1 ) [ Z:MyId <- 3 ] ) ~> .
+  </k>
```

Comment:
This is consistent with the Java backend result, meaning that `[ Y <- 2]` appears before `X |-> 1`. I believe this is a consequence of adding the simplification rule from 419 in `domains.k`. There are other tests with the same behavior. I believe that this rule application would only be visible in the test if we kept the RHS as `fail`.